src/utils/validation.ts

```diff
@@ -99,7 +95,7 @@ export function validateIGetTradesRequest(payload: IGetTradesRequest) {
   if (affiliate !== undefined && typeof affiliate !== "string") {
     return "invalid affiliate";
   }
-  if (pageSize !== undefined && typeof pageSize !== "number") {
+  if (pageSize !== undefined && !Number.isFinite(pageSize)) {
```

Comment:
perhaps `Number.isInteger`?

Reply:
Yep `Number.isInteger` will work. Also yes pageSize cannot be less than 0